### PR TITLE
Update API version, improvements in API controller.

### PR DESCRIPTION
**Pull request by kegor**
Replicated solution from https://github.com/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pull/12.

### CODE REVIEW

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 2
           

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -5
         

Clones added
============
- src/Products/Editor/Controllers/EditorApiController.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-WebForms/pullRequest?prid=5153088&bid=16924651)

File: src/Products/Common/Resources/Resources.cs [hunk 1]
```diff
@@ -63,17 +62,17 @@ public ExceptionEntity GenerateException(System.Exception ex)
         /// <param name="ex">Exception</param>
         /// <param name="password">string</param>
         /// <returns>ExceptionEntity</returns>
-        public ExceptionEntity GenerateException(System.Exception ex, String password)
+        public ExceptionEntity GenerateException(Exception ex, String password)
```
Comment (bobkovalex) [inline src/Products/Common/Resources/Resources.cs]: Codacy Issue found: [Make 'GenerateException' a static method.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-WebForms/pullRequest?prid=5153088)

File: src/Products/Common/Resources/Resources.cs [hunk 2]
```diff
@@ -47,7 +46,7 @@ public static string GetFreeFileName(string directory, string fileName)
         /// </summary>
         /// <param name="ex">Exception</param>
         /// <returns>ExceptionEntity</returns>
-        public ExceptionEntity GenerateException(System.Exception ex)
+        public ExceptionEntity GenerateException(Exception ex)
```
Comment (bobkovalex) [inline src/Products/Common/Resources/Resources.cs]: Codacy Issue found: [Make 'GenerateException' a static method.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-WebForms/pullRequest?prid=5153088)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -5
         

Clones added
============
- src/Products/Editor/Controllers/EditorApiController.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-WebForms/pullRequest?prid=5153088&bid=16924651)

**Review by Pavel-Teplitsky — APPROVED** [review]

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           

Complexity decreasing per file
==============================
+ src/Products/Editor/Controllers/EditorApiController.cs  -5
         

Clones added
============
- src/Products/Editor/Controllers/EditorApiController.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-WebForms/pullRequest?prid=5153088&bid=16924651)

**Pull request merged**